src/main/java/com/hotels/bdp/circustrain/bigquery/client/HiveTableLocationFactory.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.bigquery.client;
+
+import com.hotels.bdp.circustrain.bigquery.extraction.container.ExtractionUri;
+
+class HiveTableLocationFactory {
+  private HiveTableLocationFactory() {}
+
+  static String getLocation(ExtractionUri extractionUri) {
+    return "gs://" + extractionUri.getBucket() + "/" + extractionUri.getFolder() + "/";
```

Comment:
Maybe we could even remove this factory class entirely and have the above be a method on `ExtractionUri`?